nuxt.config.js

```diff
@@ -204,6 +204,7 @@ export default {
     dsn:
       process.env.SENTRY_DSN ||
       'https://3f3e05dbe6994c318d1bf1c8bfcf71a1@o288582.ingest.sentry.io/1525413',
+    disabled: process.env.NODE_ENV !== 'production',
```

Comment:
Nice!

Reply:
I think we'll actually want to make this _explicitly_ disabled locally somehow, because soon we'll have cloud staging and production environments and we'll want to have Sentry running on both.

Reply:
Can we have a separate var like `ENABLE_SENTRY`? It'll allow Sentry to be enabled or disabled.